projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.html

```diff
@@ -31,6 +31,12 @@
                         (click)="activateEditMode()">
                     <mat-icon>edit</mat-icon>
                 </button>
+                <button mat-button
```

Comment:
I do not understand: What information is meant here?

Reply:
here there are Creation date and Value creator. Do we need to display other kind of information?

Reply:
Ah I see...it's the creation date and value creator etc

Reply:
This information does not exist in SALSAH; Neither on the resource nor on the propert.
Here's a resource view:
https://data.dasch.swiss/resources/2632709

Reply:
> I am wondering if this information should not also be readable in read mode..

@flavens Do you mean that it should always be visible and not on a tooltip thing? I can imagine that this will be too much information when we have this information on each value. Especially if we have several values per property.

Reply:
Always on a tooltip, but visible on read and edit mode.
In read mode, only the info icon that displays a tooltip will be visible when the user mouses over a value.

Reply:
Ah I understand. Yes good idea.

Reply:
What about also having the value comment button show on hover? I just realized there is no way to see the comment in read mode currently.

Reply:
ah yes I think it is important to have it in read mode, and if I've understood correctly, it is possible that a user can have the permission to create/edit a value comment but does not have the permission to create/edit/delete a value.... but this is another issue. @kilchenmann what do you think?

Reply:
> it is possible that a user can have the permission to create/edit a value comment but does not have the permission to create/edit/delete a value.... but this is another issue.

Yes this should be solved in a separate PR.

Reply:
4d93d5e enables the property info and comment hover buttons in read mode

Reply:
> > it is possible that a user can have the permission to create/edit a value comment but does not have the permission to create/edit/delete a value.... but this is another issue.
> 
> Yes this should be solved in a separate PR.

@flavens I've created a user story for this and added it to the dsp-ui roadmap. Not sure when we'll get around to it. 
https://dasch.myjetbrains.com/youtrack/issue/DSP-683